add: main page

Ralph-1006 opened the pull request:
- login page
- main page
- landing page

Ralph-1006 committed:
initial commit

Franziska1402 commented on css/CalendarView.css:
```diff
@@ -0,0 +1,47 @@
+body {
+  color: white;
+  height: 100%;
+  font-family: "Source Code Pro";
```
mehr Font-Families

Franziska1402 commented on html/createEntry.html:
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
eventuell noch Version

Franziska1402 reviewed (changes requested):
DOCTYPE XML-konform anpassen

Ralph-1006 committed:
fuegt arial als font hinzu

Ralph-1006 committed:
footer als eigene html seite, schriftarten, backgroundcolor, ... in master.css

Ralph-1006 committed:
entfernt weitere unnoetige Angaben von Farben und Schriftarten

Crixos86 committed:
lightblue logo added

Franziska1402 committed:
Sprachen angepasst

Franziska1402 committed:
"Sprachen angepasst"

Commit:
Change: index.css background

Franziska1402 committed:
Sprachen angepasst

Franziska1402 committed:
Task als Beispiel ohne <br>

Franziska1402 committed:
"Um Typ wählen erweitert"

Commit:
Change: Landing Page

tsch4k0mo committed:
fix: display of bg image on landingpage

tsch4k0mo committed:
Change: Load XSL File into CalenderView
Fix:	Change tag to class in calenderView.cs
Delete: Hardcoded CalenderView
ToDo: 	Get XML Data into Calender

tsch4k0mo committed:
Add: 	ToDo's in Code
Fix: 	Front Page font
ToDo:	Change Butto Size on front Page

tsch4k0mo committed:
Fix: font-style in xsl file

Ralph-1006 committed:
fix: farbe von footer links, background image auf landing page wird jetzt angezeigt

tsch4k0mo committed:
Add: 	calender.xsl puts appoointments/tasks into corresponding fields
ToDo:	filter appointments/tasks by week
	filter appointments/tasks by time (no Data yet)
Danke Robin für die Dabeihilfe ;)

P3lina committed:
add: javascript code for current date
TODO: filter xml by date

tsch4k0mo committed:
Change: Move Javascript and buttons to mainPage.html
Add:	Iframe is reloaded uppon button press with date parameter
ToDo:	change url parameters on view or timechange

Franziska1402 committed:
dropdown for typ, textarea (createEntry)

tsch4k0mo committed:
Change: XML File
Add: 	Load appointments into XSL
ToDo:	Sort calendar items by time add new XML Config
